BoneReconstructionPlanner/BoneReconstructionPlanner.py

```diff
@@ -781,14 +795,14 @@ def generateFibulaPlanes(self):
         if i!=(len(planeList)-2):

```

Comment:
Starting from here

Reply:
This whole method is really, really long. A method should not be longer than what fits on 1-2 screens. Try to split it up to smaller methods.